python/cudf_polars/tests/expressions/test_evaluate_mapping.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import polars as pl
+from polars.testing import assert_frame_equal
+
+from cudf_polars.containers import DataFrame
+from cudf_polars.dsl import expr
+
+
+def test_evaluate_mapping():
```

Comment:
What is motivating these new tests? Was there missing coverage for the `mapping=` argument?

FYI: I've moved away from using the `mapping=` feature in https://github.com/rapidsai/cudf/pull/18492. So, we shouldn't need it for "complex" Select support anymore :)

Reply:
OK, if we get that in first then I will just remove the mapping argument. But previously we tested the mapping argument in grouped aggregations, having changed how we do that, we must test it separately.

Reply:
Now removed the mapping argument completely.